suricata/update/main.py

```diff
@@ -955,6 +957,10 @@ def load_sources(suricata_version):
         if not os.path.exists(index_filename):
             logger.warning("No index exists, will use bundled index.")
             logger.warning("Please run suricata-update update-sources.")
+        if os.path.exists(index_filename) and time.time() - \
+                os.stat(index_filename).st_mtime > INDEX_EXPIRATION_TIME:
+            logger.warning("Source index is greater than 2 weeks old. "
```

Comment:
"Source index is older than 2 weeks." sound much more natural to me.

Reply:
Sounds good. Will change the logger message.